v0/destinations/mailchimp/utils.js

```diff
@@ -0,0 +1,71 @@
+const axios = require("axios");
+const md5 = require("md5");
+const logger = require("../../../logger");
+const { CustomError } = require("../../util");
+
+function getMailChimpEndpoint(datacenterId, audienceId) {
```

Comment:
add documentation about this function

Reply:
We should give links to mailchimp docs for this here

Reply:
done